web/src/inbox_ui.js

```diff
@@ -174,6 +174,9 @@ function format_dm(user_ids_string, unread_count, latest_msg_id) {
         is_hidden: filter_should_hide_row({dm_key: user_ids_string}),
         is_collapsed: collapsed_containers.has("inbox-dm-header"),
         latest_msg_id,
+        mention_in_unread: unread.dms_has_mention(
+            unread.get_msg_ids_for_user_ids_string(user_ids_string),
+        ),
```

Comment:
Let's use a name that specifies that this is just for dms like `mention_in_unread_dms`.

Reply:
This needs to be done for all your changes, it make the behaviour buggy when there are mention in topics.

Reply:
yeah that makes sense, thanks. Implementing the changes